src/core/execute-serialize.c

```diff
@@ -1434,17 +1426,13 @@ static int exec_parameters_serialize(const ExecParameters *p, FILE *f, FDSet *fd
                         return r;
         }
 
-        if (p->idle_pipe) {
+        if (exec_parameters_is_idle_pipe_set(p)) {
```

Comment:
Are you sure about this? I have vague memories of seeing legitimate cases where one of the FDs was unset

Reply:
looking at the sources i don't see that.

Reply:
ok, let's wait for the CI, maybe I am misremembering